command/rubycommand/rubycommand.go

```diff
@@ -152,21 +152,48 @@ func GemUpdate(gem string) ([]*command.Model, error) {
 	return cmds, nil
 }
 
+func gemInstallCommand(gem, version string, enablePrerelease bool) []string {
+	slice := []string{"gem", "install", gem, "--no-document"}
+	if enablePrerelease {
+		slice = append(slice, "--pre")
```

Comment:
based on the [gem install docs](https://guides.rubygems.org/command-reference/#gem-install), it should be:

```
-​-[no-]prerelease - Allow prerelease versions of a gem to be installed. (Only for listed gems)
```

Reply:
It worked fine, but updated it.